docs/Administrator Guide/index.md

```diff
@@ -58,7 +56,7 @@
 
 17.  [Trash for GlusterFS](./Trash.md)
 
-18.  [Split brain and ways to deal with it](Split brain and ways to deal with it.md)
+18.  [Split brain and ways to deal with it](./Split brain and ways to deal with it.md)
```

Comment:
There are a few places like this where the link is not pointing to the file. The file might be at a different location or unavailable. Can this be looked into?

Reply:
I am aware of this have ran **markdown-link-check** on the entire repo will put up a new pull request to address this issues as it'll take some good amount of work. Would that be okay?

Reply:
Yep, you can address that as a separate PR.

Reply:
PR sent: https://github.com/gluster/glusterdocs/pull/505